src/Files.App/Dialogs/BulkRenameDialog.xaml

```diff
@@ -0,0 +1,60 @@
+﻿
+<!--  Copyright (c) 2024 Files Community. Licensed under the MIT License. See the LICENSE.  -->
+<ContentDialog
+	x:Class="Files.App.Dialogs.BulkRenameDialog"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:helpers="using:Files.App.Helpers"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	Title="{helpers:ResourceString Name=BulkRename}"
+	DefaultButton="Primary"
+	HighContrastAdjustment="None"
+	IsPrimaryButtonEnabled="{x:Bind ViewModel.IsNameValid, Mode=OneWay}"
+	PrimaryButtonCommand="{x:Bind ViewModel.CommitRenameCommand, Mode=OneWay}"
+	PrimaryButtonText="{helpers:ResourceString Name=Rename}"
+	RequestedTheme="{x:Bind RootAppElement.RequestedTheme, Mode=OneWay}"
+	SecondaryButtonText="{helpers:ResourceString Name=Cancel}"
+	Style="{StaticResource DefaultContentDialogStyle}"
+	mc:Ignorable="d">
+
+	<StackPanel Width="440" Spacing="4">
+
```

Comment:
```suggestion
```

Reply:
I already suggested changes for all of them here.

Reply:
These are different changes.

Reply:
It's a preference to have spaces between xaml blocks.

Reply:
Yes I know, this is just not the same as in other files.

Reply:
The style has evolved over the years

Reply:
> These are different changes.

I misunderstood, sorry.

Reply:
Other XAML files have this code style (end blocks have no spacing) 
When I made this code review I looked carefully into the code style of selection of random XAML files. It's just about being consistent 😅